tests/polyfill/test_h3.py

```diff
@@ -12,6 +12,16 @@ def __geo_interface__(self):
         return self.dictionary
 
 
+def same_set(a, b):
```

Comment:
it seems like this function gets duplicated a few times - maybe move it to a common test helper?

Reply:
Done